bluesky/plan_patterns.py

```diff
@@ -62,6 +62,120 @@ def spiral(x_motor, y_motor, x_start, y_start, x_range, y_range, dr, nth, *,
     cyc += cycler(y_motor, y_points)
     return cyc
 
+        
+def spiral_square_pattern(x_motor, y_motor, x_centre, y_centre, x_range, y_range, x_num,y_num):
+    '''Square spiral scan, centered around (x_start, y_start)
+    Parameters
```

Comment:
A blank line is needed above this heading to make the Sphinx documentation render correctly. Likewise for the other sections below.